application/builtin/contract/deposit/deposit.go

```diff
@@ -280,7 +276,7 @@ func (d *Deposit) canTransfer(transferAmount *big.Int) error {
 			c++
 		}
 	}
-	if d.VestingType == appfoundation.DefaultVesting && c < numConfirmation {
+	if c < numConfirmation {
```

Comment:
With this change all deposits from Genesis will be block forever, cause they don't have confirms by migration daemons.

Reply:
that's bad, will fix